contracts/bounties/BreakInvariantBounty.sol

```diff
@@ -8,24 +8,25 @@ import "../ownership/Ownable.sol";
  * @dev This bounty will pay out to a researcher if they break invariant logic of the contract.
  */
 contract BreakInvariantBounty is PullPayment, Ownable {
-  bool private _claimed;
+  bool private _claimable = true;
   mapping(address => address) private _researchers;
 
   event TargetCreated(address createdAddress);
+  event BountyCanceled(address bountyAddress);
```

Comment:
I believe this address parameter is not needed, since it will always be the address of the contract.

Reply:
So, it will be a no parameter event?

Reply:
Correct!

Reply:
done